### PR TITLE
[NOTEPAD][OSK] Improve Portuguese (Brazilian) translation (pt-BR)

### PR DESCRIPTION
**Pull request by valmorflores**
## Purpose

_Notepad, language words correction and file configuration setup ._

JIRA issue: [CORE-XXXX](https://jira.reactos.org/browse/CORE-XXXX)

## Proposed changes

_For default system settings i change rsrc.rc like: LANGUAGE_PT_BR correcting others cases and words._

- 
- 

## TODO

_Use a TODO when your pull request is Work in Progress._

- [ ] 
- [ ]

### CODE REVIEW

**Review by ericsomto — APPROVED** [review]

**Review by katahiromz — APPROVED** [review]

**Comment by valmorflores** [comment]
Hello everyone! 
I would like to know if I need to provide anything else for the publication of these changes? I would like to close the request for a new publication. Thanks!"

File: base/applications/osk/lang/pt-BR.rc [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+    BEGIN
+        MENUITEM "&Sair", IDM_EXIT
+    END
+
+    POPUP "Teclado"
+    BEGIN
+        MENUITEM "Teclado extendido", IDM_ENHANCED_KB, CHECKED
+        MENUITEM "Teclado simples", IDM_STANDARD_KB
+        MENUITEM SEPARATOR
+        MENUITEM "Layout padrao", IDM_REG_LAYOUT, CHECKED, GRAYED
```
Comment (zecarlos1957) [inline base/applications/osk/lang/pt-BR.rc:24]: A tradução para "Layout" poderá ser Estilo padrão

File: base/applications/osk/lang/pt-PT.rc [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+    BEGIN
+        MENUITEM "&Sair", IDM_EXIT
+    END
+
+    POPUP "Teclado"
+    BEGIN
+        MENUITEM "Teclado extendido", IDM_ENHANCED_KB, CHECKED
+        MENUITEM "Teclado simples", IDM_STANDARD_KB
+        MENUITEM SEPARATOR
+        MENUITEM "Layout padrao", IDM_REG_LAYOUT, CHECKED, GRAYED
```
Comment (zecarlos1957) [inline base/applications/osk/lang/pt-PT.rc:24]: Estilo padrão

File: base/applications/osk/lang/pt-PT.rc [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+        MENUITEM "Bloquear layout", IDM_BLOCK_LAYOUT, GRAYED
```
Comment (zecarlos1957) [inline base/applications/osk/lang/pt-PT.rc:25]: Bloquear estilo

File: base/applications/osk/lang/pt-PT.rc [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+        MENUITEM "101 teclas", IDM_101_KEYS, CHECKED, GRAYED
+        MENUITEM "102 teclas", IDM_102_KEYS, GRAYED
+        MENUITEM "106 teclas", IDM_106_KEYS, GRAYED
+    END
+
+    POPUP "Configurações"
+    BEGIN
+        MENUITEM "Sempre no topo", IDM_ON_TOP, CHECKED
+        MENUITEM SEPARATOR
+        MENUITEM "&Ativar som de clique", IDM_CLICK_SOUND
```
Comment (zecarlos1957) [inline base/applications/osk/lang/pt-PT.rc:36]: @Activar som de clique

File: base/applications/osk/lang/pt-PT.rc [hunk 5]
```diff
@@ -0,0 +1,84 @@
...
+        MENUITEM SEPARATOR
+        MENUITEM "&Modo digitação...", IDM_TYPE_MODE, GRAYED
+        MENUITEM SEPARATOR
+        MENUITEM "&Fonte...", IDM_FONT
```
Comment (zecarlos1957) [inline base/applications/osk/lang/pt-PT.rc:40]: Tipo de letra

File: base/applications/osk/lang/pt-PT.rc [hunk 6]
```diff
@@ -0,0 +1,84 @@
...
+END
+
+IDD_WARNINGDIALOG_OSK DIALOGEX 0, 0, 250, 97
+STYLE DS_SHELLFONT | DS_MODALFRAME | DS_CENTER | WS_POPUP | WS_CAPTION
+CAPTION "Teclado Virtual"
+FONT 8, "MS Shell Dlg", 400, 0, 0x1
+BEGIN
+    DEFPUSHBUTTON "OK", IDOK, 193, 76, 50, 14
+    ICON IDI_OSK, IDC_STATIC, 7, 17, 20, 20
+    LTEXT "Teclado Virtual fornece um nível de funcionalidade para usuários com necessidades especiais. Usuários com dificuldades motoras necessitam de um software para uso diário.", IDC_STATIC, 36, 7, 207, 33
```
Comment (zecarlos1957) [inline base/applications/osk/lang/pt-PT.rc:58]: O teclado virtual fornece um nível de funcionalidade para utilizadores com necessidades especiais. Utilizadores com dificuldades motoras necessitam de um software para uso diário.

File: base/applications/osk/lang/pt-PT.rc [hunk 7]
```diff
@@ -0,0 +1,84 @@
...
+    CONTROL "Não exibir esta mensagem novamente", IDC_SHOWWARNINGCHECK, "Button",
```
Comment (zecarlos1957) [inline base/applications/osk/lang/pt-PT.rc:59]: Não mostrar esta mensagem novamente

**Review by zecarlos1957 — CHANGES_REQUESTED** [review]

File: base/applications/osk/lang/pt-PT.rc [hunk 8]
```diff
@@ -0,0 +1,84 @@
+/*
+ * PROJECT:         ReactOS Kernel
+ * LICENSE:         GPL - See COPYING in the top level directory
+ * FILE:            base/applications/osk/lang/en-US.rc
+ * PURPOSE:         On screen keyboard (Portuguese resource).
+ * PROGRAMMERS:     Copyright Denis ROBERT
+ * TRANSLATION:     Valmor Flores (valmorflores@gmail.com)
+ */
+
+LANGUAGE LANG_PORTUGUESE, SUBLANG_DEFAULT
```
Comment (zecarlos1957) [inline base/applications/osk/lang/pt-PT.rc:10]: LANGUAGE LANG_PORTUGUESE, SUBLANG_NEUTRAL

**Review by zecarlos1957 — CHANGES_REQUESTED** [review]

File: base/applications/osk/lang/pt-PT.rc [hunk 9]
```diff
@@ -0,0 +1,84 @@
...
+    IDS_INSERT "ins"
+    IDS_NUMLOCKKEY "nlk"
+    IDS_DELETE "del"
+    IDS_END "end"
+    IDS_CTRL "ctrl"
+    IDS_LEFTALT "alt"
+    IDS_RIGHTALT "alt"
+END
+
+/* EOF */
```
Comment (zecarlos1957) [inline base/applications/osk/lang/pt-PT.rc:84]: The last line must end with '\n'. So it will be an empty line.
The same for pt-BR file.

**Review by zecarlos1957 — CHANGES_REQUESTED** [review]

File: base/applications/osk/lang/pt-PT.rc [hunk 10]
```diff
@@ -0,0 +1,84 @@
...
+    IDS_ATTN "Attn"
+    IDS_INSERT "ins"
+    IDS_NUMLOCKKEY "nlk"
+    IDS_DELETE "del"
+    IDS_END "end"
+    IDS_CTRL "ctrl"
+    IDS_LEFTALT "alt"
+    IDS_RIGHTALT "alt"
+END
+
```
Comment (zecarlos1957) [inline base/applications/osk/lang/pt-PT.rc:83]: Delete this empty line please

**Review by zecarlos1957 — CHANGES_REQUESTED** [review]

File: base/applications/osk/lang/pt-PT.rc [hunk 11]
```diff
@@ -0,0 +1,84 @@
...
+        MENUITEM SEPARATOR
+        MENUITEM "Layout padrao", IDM_REG_LAYOUT, CHECKED, GRAYED
+        MENUITEM "Bloquear layout", IDM_BLOCK_LAYOUT, GRAYED
+        MENUITEM SEPARATOR
+        MENUITEM "101 teclas", IDM_101_KEYS, CHECKED, GRAYED
+        MENUITEM "102 teclas", IDM_102_KEYS, GRAYED
+        MENUITEM "106 teclas", IDM_106_KEYS, GRAYED
+    END
+
+    POPUP "Configurações"
```
Comment (zecarlos1957) [inline base/applications/osk/lang/pt-PT.rc:32]: Definições

**Review by zecarlos1957 — CHANGES_REQUESTED** [review]

File: base/applications/osk/lang/pt-PT.rc (hunk 8 above)
Comment (valmorflores) [reply base/applications/osk/lang/pt-PT.rc]: Thank you. I was in doubt, as there were files with both forms.

File: base/applications/osk/lang/pt-PT.rc (hunk 10 above)
Comment (valmorflores) [reply base/applications/osk/lang/pt-PT.rc:83]: Ok, thanks!

**Review by zecarlos1957 — APPROVED** [review]

**Review by julenuri — CHANGES_REQUESTED** [review]

**Review by zecarlos1957 — APPROVED** [review]

**Review by julenuri — APPROVED** [review]

File: base/applications/notepad/rsrc.rc [hunk 12]
```diff
@@ -110,7 +110,7 @@ IDI_NPICON ICON "res/notepad.ico"
 #ifdef LANGUAGE_PT_PT
     #include "lang/pt-PT.rc"
 #endif
-#ifdef LANGUAGE_PT_PB
+#ifdef LANGUAGE_PT_BR
     #include "lang/pt-BR.rc"
 #endif
```
Comment (HBelusca) [inline base/applications/notepad/rsrc.rc:115]:
```suggestion
```

File: base/applications/notepad/rsrc.rc [hunk 13]
```diff
@@ -110,7 +110,7 @@ IDI_NPICON ICON "res/notepad.ico"
 #ifdef LANGUAGE_PT_PT
```
Comment (HBelusca) [inline base/applications/notepad/rsrc.rc:110]:
```suggestion
#ifdef LANGUAGE_PT_BR
    #include "lang/pt-BR.rc"
#endif
#ifdef LANGUAGE_PT_PT
```

File: base/applications/osk/lang/pt-BR.rc [hunk 14]
```diff
@@ -0,0 +1,82 @@
...
+    ICON IDI_OSK, IDC_STATIC, 7, 17, 20, 20
+    LTEXT "O Teclado Virtual fornece um nível de funcionalidade para usuários com necessidades especiais. Usuários com dificuldades motoras necessitam de um software para uso diário.", IDC_STATIC, 36, 7, 207, 33
+    CONTROL "Não exibir esta mensagem novamente", IDC_SHOWWARNINGCHECK, "Button",
+            BS_AUTOCHECKBOX | WS_TABSTOP, 43, 80, 137, 10
+END
+
+STRINGTABLE
+BEGIN
+    IDS_OSK "Teclado Virtual"
+    IDS_AUTHORS "Copyright Denis Robert, Valmor Flores"
```
Comment (JoachimHenze) [inline base/applications/osk/lang/pt-BR.rc:64]: It is not a good idea to add a translator within the IDS_AUTHORS.

File: base/applications/osk/lang/pt-BR.rc [hunk 15]
```diff
@@ -0,0 +1,82 @@
...
+    IDS_INSERT "ins"
+    IDS_NUMLOCKKEY "nlk"
+    IDS_DELETE "del"
+    IDS_END "end"
+    IDS_CTRL "ctrl"
+    IDS_LEFTALT "alt"
+    IDS_RIGHTALT "alt"
+END
+
+/* EOF */
```
Comment (HBelusca) [inline base/applications/osk/lang/pt-BR.rc:82]:
```suggestion
/* EOF */

```

File: base/applications/osk/lang/pt-PT.rc [hunk 16]
```diff
@@ -0,0 +1,82 @@
...
+    IDS_INSERT "ins"
+    IDS_NUMLOCKKEY "nlk"
+    IDS_DELETE "del"
+    IDS_END "end"
+    IDS_CTRL "ctrl"
+    IDS_LEFTALT "alt"
+    IDS_RIGHTALT "alt"
+END
+
+/* EOF */
```
Comment (HBelusca) [inline base/applications/osk/lang/pt-PT.rc:82]:
```suggestion
/* EOF */

```

File: base/applications/osk/lang/pt-PT.rc [hunk 17]
```diff
@@ -0,0 +1,82 @@
...
+    ICON IDI_OSK, IDC_STATIC, 7, 17, 20, 20
+    LTEXT "O Teclado Virtual fornece um nível de funcionalidades para utilizadores com necessidades especiais. Utilizadores com dificuldades motoras necessitam de um software para uso diário.", IDC_STATIC, 36, 7, 207, 33
+    CONTROL "Não mostrar esta mensagem novamente", IDC_SHOWWARNINGCHECK, "Button",
+            BS_AUTOCHECKBOX | WS_TABSTOP, 43, 80, 137, 10
+END
+
+STRINGTABLE
+BEGIN
+    IDS_OSK "Teclado Virtual"
+    IDS_AUTHORS "Copyright Denis Robert, Valmor Flores"
```
Comment (JoachimHenze) [inline base/applications/osk/lang/pt-PT.rc:64]: It is not a good idea to add a translators name to the IDS_AUTHORS.

File: base/applications/osk/lang/pt-BR.rc (hunk 14 above)
Comment (HBelusca) [inline base/applications/osk/lang/pt-BR.rc:64]:
```suggestion
    IDS_AUTHORS "Copyright Denis Robert"
```

File: base/applications/osk/lang/pt-PT.rc (hunk 17 above)
Comment (HBelusca) [inline base/applications/osk/lang/pt-PT.rc:64]:
```suggestion
    IDS_AUTHORS "Copyright Denis Robert"
```

**Pull request merged**